scripts/ci/selective_ci_checks.sh

```diff
@@ -475,8 +475,11 @@ function get_count_kubernetes_files() {
     echo
     local pattern_array=(
         "^airflow/kubernetes"
+        "^airflow/executors/kubernetes_executor.py"
         "^chart"
-        "^tests/kubernetes_tests"
+        "^tests/kubernetes"
+        "^kubernetes_tests"
+        "^tests/executors/test_kubernetes_executor.py"
```

Comment:
```suggestion
```
Same here.

Reply:
Changes to this file should run the Kube "system" tests